### PR TITLE
Remove chromy as per discussion in https://github.com/garris/BackstopJS/issues/1180

### PR DESCRIPTION
**Pull request by broksonic21**
This is my first commit in BackstopJS repo, so apologies in advance if over or under-aggressive on this. Happy to tweak up.

This removes all references to chromy I could find as per discussion in https://github.com/garris/BackstopJS/issues/1180

One test case json came out different in the smoke tests; I think as I changed one test from chromy to puppeteer. Let me know if that should be reverted.

Note: While this addresses the chromy `npm audit` findings, I did not run `npm audit --fix` when installing locally for this PR, but I noticed a number of alerts in the devDependencies around this.

### CODE REVIEW

File: core/util/createBitmaps.js [hunk 1]
```diff
@@ -133,17 +127,7 @@ function delegateScenarios (config) {
...
-    var getFreePorts = require('./getFreePorts');
-    return getFreePorts(PORT, scenarioViews.length).then(freeports => {
-      console.log('These ports will be used:', JSON.stringify(freeports));
-      scenarioViews.forEach((scenarioView, i) => {
-        scenarioView.assignedPort = freeports[i];
-      });
-      return pMap(scenarioViews, runChromy, { concurrency: asyncCaptureLimit });
-    });
-  } else if (config.engine.startsWith('puppet')) {
+  if (config.engine.startsWith('puppet')) {
```
Comment (garris) [inline core/util/createBitmaps.js:130]: Could we leave the `/chrom./` case and throw and error here -- something like -- _Chromy is no longer supported in version 5+.  Please use version 4.x.x for chromy support._

Comment (ahaverbuch) [reply core/util/createBitmaps.js:130]: updated!

**Comment by ahaverbuch** [comment]
I ran smoke-tests, sanity-test (similar errors, only file updated looks just due to system difference and is included here) and unit-test (all passed). Integration-tests didn't run for me.

Is there more needed?

File: core/util/createBitmaps.js (hunk 1 above)
Comment (broksonic21) [reply core/util/createBitmaps.js:130]: updated!

**Comment by broksonic21** [comment]
I ran smoke-tests, sanity-test (similar errors, only file updated looks just due to system difference and is included here) and unit-test (all passed). Integration-tests didn't run for me.

Is there more needed?

File: core/util/createBitmaps.js [hunk 2]
```diff
@@ -129,6 +129,9 @@ function delegateScenarios (config) {
 
   if (config.engine.startsWith('puppet')) {
     return pMap(scenarioViews, runPuppet, { concurrency: asyncCaptureLimit });
+  }
+  else if (/chrom./i.test(config.engine)) {
+    logger.error(`Engine "${(typeof config.engine === 'string' && config.engine) || 'undefined'}" not recognized! Chromy is no longer supported in version 5+. Please use version 4.x.x for chromy support.`);
```
Comment (garris) [inline core/util/createBitmaps.js:134]: Maybe this should just be: "Chromy is no longer supported in version 5+. Please use version 4.x.x for Chromy support."  I don't think you need the first bit with all the type checking and stuff.  We already know inside this block that we have a chromy user who probably bumped their version and now they are confused about why everything is failing.  Please let me know if there is something I am missing here.  Thank you!

Comment (broksonic21) [reply core/util/createBitmaps.js]: updated

**Comment by broksonic21** [comment]
Reverted jsonReport and updated error message. Once this merges, happy to make a follow-on PR to get rid of some of the local `npm audit` issues seen (mostly in dev dependencies) that just need updating. Didn't include here for separation of concern reasons.

**Comment by garris** [comment]
Looks great -- thanks for all the deleting -- I feel so much lighter now!

**Pull request merged**